velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -122,23 +131,26 @@ std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
   // k RANGE frames require more work as we have to generate columns with the
   // frame bound values.
   std::vector<core::WindowNode::BoundType> startBoundOptions, endBoundOptions;
-  if (isRowsFrame) {
+  // If it is a range frame and sorting keys size is not 1, then that means
+  // k-range frames should not be allowed for random selection
+  if (typeOfFrame == core::WindowNode::WindowType::kRange &&
```

Comment:
This is a bit unreadable. The following order of conditionals is easier to understand.

if (rows frame) {
... all the options here...
} else {
// This is a range frame
if (kRange) {
  ...options here...
} else {
... non k-range frame options...
}
}